python/feathub/processors/local/local_processor.py

```diff
@@ -611,3 +612,7 @@ def _get_dependent_features(self, feature_view: FeatureView) -> Sequence[Feature
             if feature not in dependent_features:
                 dependent_features.append(feature)
         return dependent_features
+
+    def _filter_df(self, df: pd.DataFrame, filter_expr: str) -> pd.DataFrame:
```

Comment:
Would it be better to rename this method to `_filter_dataframe`? I noticed that none of the other methods use abbreviations in its signature.